src/compiler/diagnosticMessages.json

```diff
@@ -1924,5 +1924,9 @@
     "The 'arguments' object cannot be referenced in an arrow function. Consider using a standard function expression.": {
         "category": "Error",
         "code": 9002
+    },
+    "Tuple type '{0}' with length '{1}' can't be assigned to tuple with length '{2}'.": {

```

Comment:
I'm not sure what our policy is with contractions in error messages (i.e. "can't" -> "cannot").

@JsonFreeman can make a judgement on whether there should be a singular article here; I'm not particularly picky about it in this context since I think "A tuple type" would sound awkward.